File: Tier1-Blazor/Tier1/Domain/Models/ElectricityUsage.cs
namespace Domain.Models;

public class ElectricityUsage
{
    private int id { get; }
    private double amount { get; }
    private int month { get; }
    private int year { get; }
    private int userId { get; }

    public ElectricityUsage(int id,double amount, int month, int year, int userId)
    {
        this.id = id;
        this.amount = amount;
        this.month = month;
        this.year = year;
        this.userId = userId;
    }
}

File: Tier2-JavaSpring/tier2/src/main/java/via/sep3/tier2/model/ElectricityUsage.java
package via.sep3.tier2.model;

public class ElectricityUsage {

    private int id;
    private int amount;
    private int month;
    private int year;

    public ElectricityUsage(){}
    public ElectricityUsage(int amount, int month, int year, int userId) {
        this.amount = amount;
        this.month = month;
        this.year = year;
        this.userId = userId;
    }

    private int userId;

    public ElectricityUsage(int id, int amount, int month, int year, int userId)
    {
        this.id = id;
        this.amount = amount;
        this.month = month;
        this.year = year;
        this.userId = userId;
    }

    public int getId()
    {
        return id;
    }

    public void setId(int id)
    {
        this.id = id;
    }

    public double getAmount()
    {
        return amount;
    }

    public void setAmount(int amount)
    {
        this.amount = amount;
    }

    public int getMonth()
    {
        return month;
    }

    public void setMonth(int month)
    {
        this.month = month;
    }

    public int getYear()
    {
        return year;
    }

    public void setYear(int year)
    {
        this.year = year;
    }

    public int getUserId()
    {
        return userId;
    }

    public void setUserId(int userId)
    {
        this.userId = userId;
    }


}
